scripts/remote/remote-upload-output.sh

```diff
@@ -17,6 +17,7 @@ keyfile=${3:-$HOME/.ssh/ecs-key.pem}
 user=${4:-ec2-user}
 remote_variables_file=${5:-/home/ec2-user/agief-project/variables/variables-ec2.sh}
 port=${6:-22}
+no_compress=${7:False}
```

Comment:
Why create a `no_compress` variable instead of a `compress` variable?
It results in double negatives so can be a bit confusing?
Is it because `compress` should be the default state? That is a good reason.